pandas/io/parsers.py

```diff
@@ -3128,10 +3094,8 @@ def _rows_to_cols(self, content):
 
             for row_num, actual_len in bad_lines:
                 msg = (
-                    "Expected {col_len} fields in line {line}, saw "
-                    "{length}".format(
-                        col_len=col_len, line=(row_num + 1), length=actual_len
-                    )
+                    f"Expected {col_len} fields in line {row_num + 1}, saw "
+                    "{actual_len}"
```

Comment:
```suggestion
                    f"{actual_len}"
```
this needs to be a f-string